### DISCUSSION TITLE
Integration Log Warning

### DISCUSSION THREAD

**Discussion by Happy-Cadaver** [blakeblackshear/frigate#14165] [Ask A Question] [answered]
I just upgraded to HAOS 24.10.1 and got a new warning about Frigate integration in my logs. I didn't see it in 'Issues', 'Pull requests', or 'Discussions' so I figured I'd post it here in case it's being missed.

`Logger: homeassistant.components.media_player.const
Source: helpers/deprecation.py:197
integration: Media player (documentation, issues)
First occurred: 4:03:26 PM (6 occurrences)
Last logged: 4:03:26 PM

MEDIA_CLASS_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.IMAGE instead, please report it to the author of the 'frigate' custom integration
MEDIA_CLASS_MOVIE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.MOVIE instead, please report it to the author of the 'frigate' custom integration
MEDIA_CLASS_VIDEO was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.VIDEO instead, please report it to the author of the 'frigate' custom integration
MEDIA_TYPE_IMAGE was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.IMAGE instead, please report it to the author of the 'frigate' custom integration
MEDIA_TYPE_VIDEO was used from frigate, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.VIDEO instead, please report it to the author of the 'frigate' custom integration`

**Comment by hawkeye217** [comment, marked as answer]
This github repo is for Frigate itself. This has already been reported on the integration github repo and fixed: https://github.com/blakeblackshear/frigate-hass-integration/pull/726

**Reply by Happy-Cadaver** [reply]
Thank you and my appoligies, I did not know where to look..

**Reply by hawkeye217** [reply]
No worries, appreciate the report.
